### DISCUSSION TITLE
Webpage settings --> store DESKTOP and DARK settings

### DISCUSSION THREAD

**Discussion by andybehre** [Slion/Fulguris#594] [Q&A]
Hello Slion,

like your browser very much! Great functions and GUI.
Whould suggest to store the web page settings for DESKTOP and DARK-mode. For now user has to tick every time the site is visited.

I'm using the Android version of fulguris v1.9.17

Best regards Andreas

**Comment by Slion** [comment]
You need to use domain settings for it to persist.
Menu / Options / Domain
